app/index.html

```diff
@@ -19,6 +19,7 @@
 
     <script src="assets/vendor.js"></script>
     <script src="assets/super-rentals.js"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?v=3.23"></script>
```

Comment:
This is the best way, apart from using the inline-content addon, which is probably too much here.